src/app/components/ForecastCard/ForecastCard.module.css

```diff
@@ -0,0 +1,26 @@
+.forecastCard {
+  font-size: 2em;
+  color: var(--clr-text-secondary);
+  border-radius: 10px;
+  background: #9dc3e230;
```

Comment:
In Line 3 you are using a color variable. And here not. Would it be a good idea to use it here as well?